work14/index.js

```diff
@@ -25,34 +32,28 @@ const url = "https://mocki.io/v1/d6da0b8a-3546-419e-aaec-05f3a247c6d0";
 //   ]
 // }
 
-async function init() {
-  renderLoading();
-  const data = await getListData();
-  renderListElement(data);
-}
-
 async function getData() {
   try {
     const response = await fetch(url);
     if (response.ok) {
       const json = await response.json();
       return json;
-    } else {
-      throw new Error(`Server request failed:${response.statusText}`);
+    } else if (response.status >= 400) {
+      console.error(response.statusText);
+      wrap.textContent = `${"ページは存在しないようです：" + response.status}`;
     }
     // 下記は、固定値をそのままpromiseの返り値とする
     // const json = url
     // return json
   } catch (e) {
-    console.error(e)
+    console.error(e);
     throw new Error(e);
   }
 }
```

Comment:
IMO
I don't think the error message "ページは存在しないようです" is appropriate.
Because the 400th migration error is not just the error that the page does not exist.

Reply:
I was wrong. "The page doesn't seem to exist" was a 404 error.
thank you.

commit: [b9c90f3](https://github.com/yasu-pro/js-handson/pull/16/commits/b9c90f36eee2d9eac6cbd45d3ee3682852e8ea68)

Reply:
Thank you for the fix.
I also learned!